src/main/java/roomescape/infrastructure/JdbcTimeRepository.java

```diff
@@ -0,0 +1,97 @@
+package roomescape.infrastructure;
+
+import java.sql.Time;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.ReservationTime;
+import roomescape.domain.repository.TimeRepository;
+import roomescape.domain.repository.dto.TimeDataWithBookingInfo;
+
+@Repository
+public class JdbcTimeRepository implements TimeRepository {
+
+    private final static RowMapper<ReservationTime> RESERVATION_TIME_ROW_MAPPER =
+            (rs, rowNum) -> ReservationTime.of(
+                    rs.getLong("id"),
+                    rs.getTime("start_at").toLocalTime()
+            );
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public JdbcTimeRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public Long save(ReservationTime reservationTime) {
+        SimpleJdbcInsert simpleJdbcInsert = new SimpleJdbcInsert(jdbcTemplate)
+                .withTableName("reservation_time")
+                .usingGeneratedKeyColumns("id");
+
+        Map<String, Object> params = Map.of(
+                "start_at", Time.valueOf(reservationTime.getStartAt())
+        );
+
+        Number key = simpleJdbcInsert.executeAndReturnKey(params);
+        return key.longValue();
+    }
+
+    @Override
+    public List<ReservationTime> findAll() {
+        String findAllSql = "SELECT id, start_at FROM reservation_time";
+
+        return jdbcTemplate.query(findAllSql, RESERVATION_TIME_ROW_MAPPER);
+    }
+
+    @Override
+    public Optional<ReservationTime> findById(Long id) {
+        String selectOneSql = "SELECT id, start_at FROM reservation_time WHERE id=?";
+        try {
+            ReservationTime time = jdbcTemplate.queryForObject(selectOneSql, RESERVATION_TIME_ROW_MAPPER,
+                    id);
+            return Optional.of(time);
+        } catch (EmptyResultDataAccessException e) {
+            return Optional.empty();
+        }
+    }
+
+    @Override
+    public boolean deleteById(Long id) {
+        String deleteSql = "DELETE FROM reservation_time WHERE id=?";
+
+        boolean updated = jdbcTemplate.update(deleteSql, id) > 0;
+        return updated;
```

Comment:
변수를 바로 리턴한다면 할당하지 않고 바로 리턴하는 것이 좀 더 간결합니다.

```suggestion
        return jdbcTemplate.update(deleteSql, id) > 0;
```

Reply:
바로 리턴하는 방식으로 수정했습니다 :)